lld/test/ELF/systemz-plt.s

```diff
@@ -0,0 +1,73 @@
+// REQUIRES: systemz
```

Comment:
Test more symbols like `riscv-plt.s`

Reply:
Done.   This actually uncovered a bug - it was possible for `.got` to be removed completely in a file that uses `.plt` - but the PLT header refers to `_GLOBAL_OFFSET_TABLE_` which points to the start of `.got` on SystemZ ...   Fixed by enforcing the presence of `.got` in `SystemZ::addPltHeaderSymbols`.